### PR TITLE
bt-189: Not found page

### LINKED ISSUE #189: Not found page
Use our design to make some 404 page

**Acceptance criteria:**
- The page should have the BSA Talents logo
- The page should have a response status code and a short description of the error
- The page should have a link to go back to the main page
- The page should have an animation

### PR DESCRIPTION
**Pull request by Monyster**
Closes #189 

If you route to the wrong link, you will see:
image
 
Also created page layout:
image

Example of code:
  ```js
<PageLayout avatarUrl={''} isOnline>
                {[...Array(100)].map((_, index) => (<div key={index}> <br />Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea inventore, architecto ipsum ad a distinctio possimus! Fuga architecto, exercitationem reprehenderit ipsam provident maxime quod blanditiis eligendi dolore eaque dignissimos doloremque?</div>))}
</PageLayout >
```

### CODE REVIEW

File: frontend/src/bundles/common/components/page-layout/styles.module.scss
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/page-layout/styles.module.scss:1]: @Monyster contact with @Simpsonkk about page layout

File: frontend/src/bundles/common/components/page-layout/page-layout.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    isOnline: boolean;
+    children: React.ReactNode;
+};
+
+const PageLayout: React.FC<Properties> = ({
+    avatarUrl,
+    isOnline,
+    children,
+}) => (
+    <div className={getValidClassNames(styles.pageContainer)}>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/page-layout/page-layout.tsx]: Better to use Grid component

File: frontend/src/bundles/common/pages/not-found/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Link, Logo, Typography } from '../../components/components.js';
+import { AppRoute } from '../../enums/app-route.enum.js';
+import { getValidClassNames } from '../../helpers/helpers.js';
+import styles from './styles.module.scss';
+
+const NotFoundPage: React.FC = () => (
+    <div className={getValidClassNames(styles.pageContainer)}>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/pages/not-found/not-found.tsx]: looks like we can use Grid component

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/page-layout/styles.module.scss
Comment (Simpsonkk) [reply frontend/src/bundles/common/components/page-layout/styles.module.scss:1]: Yep, for your current layout, you don't need a page layout therefore you can remove it here.

**Comment by Simpsonkk** [comment]
The page looks great on my local machine too. Why did you change the position header to sticky? I think we should use fixed but I see that you have not already used the header on your page and probably will remove these changes.

**Comment by Monyster** [comment]
> The page looks great on my local machine too. Why did you change the position header to sticky? I think we should use fixed but I see that you have not already used the header on your page and probably will remove these changes.

When you use :fixed, it applies top, left, etc. to global window, but if you use :sticky, you can move it to any parent component with position:relative;
Thus, we have sidebars that cross the same level as thwe header. And we can to hide some stuffs in header.

To solve this, did next:
- The left side is the sidebar and the right side is the [container].
- The [container] has a position:relative.
- And the header is aligned inside this [container] without hiding behind the sidebar.

and when you pass child element to it, it already places in the prepared container, not on the global page

image

**Comment by Monyster** [comment]
I changed :sticky to :fixed and change styles for page layout.

**Review by Simpsonkk — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**